ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.js

```diff
@@ -22,6 +22,7 @@
       id: 'Brayflox Normal Pelican Poison Healer',
       netRegex: NetRegexes.gainsEffect({ effectId: '12' }),
       condition: (data) => data.role === 'healer',
+      suppressSeconds: 2,
```

Comment:
I think this `suppressSeconds` works opposite of how the comment above explains how this trigger should work.
By suppressing here, won't you only see this trigger once for Player A, even if Players A, B, and C all got hit?

For the same effect, you'd need one trigger to collect the people who got hit without a message, and another trigger (with suppressSeconds) to display the message for everyone that got hit.

I would expect something similar to:
```js
{
  id: 'Brayflox Normal Pelican Poison Collect',
  netRegex: NetRegexes.gainsEffect({ effectId: '12' }),
  condition: (data) => data.role === 'healer',
  run: (data, matches) => {
    data.pelicanPoisons = data.pelicanPoisons || [];
    data.pelicanPoisons.push(matches.target);
  }
},
{
  id: 'Brayflox Normal Pelican Poison Healer',
  netRegex: NetRegexes.gainsEffect({ effectId: '12' }),
  condition: (data) => data.role === 'healer',
  delaySeconds: 1,
  suppressSeconds: 2,
  alertText: (data, matches) => {
    if (!data.pelicanPoisons)
      return;

    const names = data.pelicanPoisons.sort();
    data.pelicanPoisons = [];
    if (names.length === 1 && names[0] === data.me) {
      return {
        en: 'Esuna Your Poison',
        de: 'Entferne dein Gift',
        fr: 'Purifiez-vous',
        cn: '康复自己的毒',
      };
    }
    return {
      en: 'Esuna Poison on ' + names.map((x) => data.ShortName(x)).join(', '),
      de: 'Entferne Gift von ' + names.map((x) => data.ShortName(x)).join(', '),
      fr: 'Purifiez le poison sur ' + names.map((x) => data.ShortName(x)).join(', '),
      cn: '康复' + names.map((x) => data.ShortName(x)).join(', '), + '的毒',
    };
  },
},
```

Reply:
You are correct, I needed to update this because during my daily I had the trigger go off 3 times in quick succession on the tank. Thank you for the input.